tests/check/signed_video_helpers.h

```diff
@@ -32,13 +32,16 @@
 #define MANUFACT "manufacturer"
 #define ADDR "address"
 
+typedef enum { SV_RECURRENCE_DEFAULT = 1, SV_RECURRENCE_THREE = 3} SignedVideoRecurrence;
```

Comment:
Maybe we should have three variants; SV_NO_RECURRENCE, SV_RECURRENCE_THREE, SV_RECURRENCE_THREE_WITH_OFFSET